tensorflow/contrib/cmake/tf_cc_ops.cmake

```diff
@@ -155,7 +155,7 @@ if (WIN32)
     set (pywrap_tensorflow_lib "${CMAKE_CURRENT_BINARY_DIR}/pywrap_tensorflow_internal.lib")
   endif()
 else (WIN32)
-  set (pywrap_tensorflow_lib "${CMAKE_CURRENT_BINARY_DIR}/libpywrap_tensorflow_internal.so")
+  set (pywrap_tensorflow_lib "${CMAKE_CURRENT_BINARY_DIR}/libpywrap_tensorflow_internal${CMAKE_SHARED_LIBRARY_SUFFIX}")
```

Comment:
On Mac, this shared lib compiled as libpywrap_tensorflow_internal.dylib. Since the shared lib extension name depends on platform, it's better to use ${CMAKE_SHARED_LIBRARY_SUFFIX}. 
Error message: 
[100%] Linking CXX shared library libpywrap_tensorflow_internal.dylib
[100%] Built target pywrap_tensorflow_internal
make[2]: *** No rule to make target `libpywrap_tensorflow_internal.so', needed by `lib_beam_search_ops.dylib'.  Stop..